README.md

```diff
@@ -517,14 +517,53 @@ Specifies the file extension for **local** script files in your Apps Script proj
 
 Specifies the files that should be pushed first, useful for scripts that rely on order of execution. All other files are pushed after this list of files.
 
+## Using clasp as a local dependency
```

Comment:
I don't support this on the main README.

Create a separate `doc/` page if we want to add this guide.

Reply:
Will do